cross-tests/src/test/java/org/apache/iotdb/cross/tests/tools/importCsv/ExportCsvTestIT.java

```diff
@@ -46,6 +46,22 @@
 
   private final String EXPORT_FILE = "target" + File.separator + "dump0.csv";
 
+  private final String[] outputForWindows = {
+    "------------------------------------------",
+    "Starting IoTDB Client Export Script",
+    "------------------------------------------",
+    "Start to export data from sql statement",
+    "successfully",
+  };
+
+  private final String[] outputForUnix = {
+    "------------------------------------------",
+    "Starting IoTDB Client Export Script",
+    "------------------------------------------",
+    "Start to export data from sql statement",
+    "successfully",
+  };
+
```

Comment:
Can we combine the `outputForWindows` and `outputForUnix` together? as I see the string arrays are the same.

And the same as others.